packages/@react-aria/utils/src/platform.ts

```diff
@@ -11,27 +11,31 @@
  */
 
 function testUserAgent(re: RegExp) {
-  return typeof window !== 'undefined' && window.navigator != null
-    ? re.test(window.navigator.userAgent)
-    : false;
+  if (typeof window === 'undefined' || window.navigator == null) {
+    return false;
+  }
+  return (
+    window.navigator['userAgentData']?.brands.some((brand: {brand: string, version: string}) => re.test(brand.brand))
```

Comment:
Note: `userAgentData` only is only supported in HTTPS? https://developer.mozilla.org/en-US/docs/Web/API/Navigator/userAgentData

Reply:
isn't that fine? we prefer to use it, but fall back if not?